sql/moz-fx-data-shared-prod/accounts_db_external/fxa_oauth_clients_v1/query.sql

```diff
@@ -1,5 +1,5 @@
 SELECT
-  id,
+  TO_HEX(id) AS id,
```

Comment:
The `accounts_db.fxa_oauth_clients` view is currently calling `TO_HEX()` on this table's `id` column, so this PR needs to update that view not to do that or the view query will fail with an error.